tests/run/pep526_variable_annotations.py

```diff
@@ -161,13 +158,58 @@ def literal_list_ptr():
     return a[3]
 
 
+def test_subscripted_types():
+    """
+    >>> test_subscripted_types()
+    dict object
+    list object
+    set object
+    """
+    a: typing.Dict[int, float] = {}
+    b: List[int] = []
+    c: _SET_[object] = set()
+
+    print(cython.typeof(a) + (" object" if not cython.compiled else ""))
+    print(cython.typeof(b) + (" object" if not cython.compiled else ""))
+    print(cython.typeof(c) + (" object" if not cython.compiled else ""))
+
+# because tuple is specifically special cased to go to ctuple where possible
+def test_tuple(a : typing.Tuple[int, float], b : typing.Tuple[int, ...],
+               c : Tuple[int, object]  # cannot be a ctuple
+               ):
```

Comment:
I'm arguing that this is covered by

> The closing brace/bracket/parenthesis on multiline constructs may either line up under the first non-whitespace character of the last line of list [...]

of [PEP 8](https://www.python.org/dev/peps/pep-0008/).

I agree it's a little odd though - I think this was the best option that let me have a comment immediately following the argument.

Reply:
The indent itself is fine - that's a visual indent (not my preference but it works). - what I am referring to is the spaces around the colon. Should it not be `a: XYZ` instead of `a : XYZ`?

Reply:
Yes - you're right. Thanks.

I'll fix that too locally and push it next time I properly update it.